index.bs

```diff
@@ -2246,6 +2249,13 @@ the [RP] know that the extension was honored by the client. Similarly, any exten
 specify a method of augmenting the [=authenticator data=] to let the [RP] know that the extension was honored by the
 authenticator.
 
+If an extension is designed to enable the [RP] to know whether it was understood and acted on,
+the extension MUST return some data from the client to the [RP] for this purpose.
+If an extension does not otherwise require any result values, it SHOULD be defined as returning a Boolean extension result,
+set to `true` to signify that the extension was understood and processed.
+Likewise, such an extension that is processed by the authenticator that does not otherwise require any result values
+SHOULD also return a Boolean extension result,
+set to `true` to signify that the extension was understood and processed.
```

Comment:
The choice of "true" as the suggested default extension result was to be parallel with the choice of "true" as the client argument value at https://w3c.github.io/webauthn/#extension-request-parameters.  It would be puzzling to use different conventions in the same cases when the reason for these default values is the same.